docs/15-deploy-to-localstack.md

```diff
@@ -0,0 +1,42 @@
+Building and deploying Sleeper locally
+======================================
+
+You can also run a localstack container locally and deploy an instance of Sleeper to it. This deployment method has limited
+functionality, but will allow you to peform a queue-based standard ingest, and run compactions on a single table.
+
+## Launch Localstack container
+To launch the localstack container, run the `startContainer.sh` script in the `scripts/deploy/localstack` folder.
```

Comment:
This needs docker-compose to be installed - this is worth documenting.

If you run `./scripts/deploy/localstack/startContainer.sh` from inside the top level Sleeper directory, it fails with `/home/ec2-user/sleeper-main/docker-compose.yml: no such file or directory`. It works if you run it from inside `scripts/deploy/localstack/`.